{
  setFailFast(true);
  try {
    eval(input);
    fail();
  }
 catch (  IllegalArgumentException|EvalException e) {
    assertThat(e).hasMessage(msg);
  }
}
